resources/pages/RequestedSongs.jsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Button, Stack, Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import PlaylistCard from "../components/PlaylistCard";
+
+const RequestedSongs = () => {
+    const [requestedSongs, setRequestedSongs] = useState([]);
+    const { id } = useParams();
+
+    useEffect(() => {
+        fetch(`http://playlist-app.localhost/api/companies/${id}/songs`)
```

Comment:
Essaye de réutiliser l'APP_URL qui est déjà dans ton .env laravel : https://www.itsolutionstuff.com/post/how-to-get-environment-variable-in-laravel-react-jsexample.html

Reply:
Create VITE_APP_URL in .env `VITE_APP_URL="${APP_URL}/api"`
Added `export const apiUrl = import.meta.env.VITE_APP_URL;` in App.js 
Replaced with`fetch(`${apiUrl}/companies/${id}/songs`)`